x-pack/legacy/plugins/ml/public/util/index_utils.ts

```diff
@@ -62,7 +46,7 @@ export function getIndexPatternIdFromName(name: string) {
       return indexPatternCache[j].id;
     }
   }
-  return name;
+  return null;
```

Comment:
Just to clarify, this is changing the default return type to `null` rather than returning the `name` for cases where no matching index pattern could be found. Probably makes more sense, but just want to make sure!

Reply:
i checked the places where this is called and returning `null` makes more sense.
After this refactor, there are no places where returning the originally supplied index name is expected.